### ISSUE TITLE
[19w29b][BUG] Rotator rarely reaching 0 degrees

### ISSUE THREAD

**Issue by Tommyg930** [tsunamayo/Skywanderers#1388]
### Short Description of the issue:

while working on a ship with hanger doors that use the Rotator as a mechanism, I've noticed that the Rotator  rarely reaches its correct resting position. no matter if its manually controlled or sequencer controlled. it always seems to be off by about 0.1 to 0.4 degrees, which becomes very noticeable the further the blocks sticks out from the rotational axis.

### Environment
- Skywanderers version: 19w29b
- Graphics Card: RTX 2080

**Comment by tsunamayo** [comment]
Yes I can understand why. It would have to wait for the new collision system as collision is CPU intensive. I will keep you posted. Cheers
